setup.py

```diff
@@ -47,6 +44,7 @@
     classifiers=[
         'Development Status :: 4 - Beta',
         'Intended Audience :: Developers',
+        'License :: OSI Approved :: MIT License',
```

Comment:
The work is under the ["MIT" license](https://opensource.org/licenses/MIT) (more precisely, Expat), so this is correct.